monitoring/README.md

```diff
@@ -15,4 +15,222 @@ This directory contains chaos interleaved grafana dashboards to get started with
+
+- Import the grafana dashboard "Node and Pod Chaos Demo" provided [here](https://raw.githubusercontent.com/litmuschaos/litmus/master/monitoring/grafana-dashboards/kubernetes/Node-and-pod-metrics-dashboard.json)
+
+### Step-4: Execute the Chaos Experiments
+
+- For the sake of illustration, let us execute node and pod level, CPU hog experiments on the `catalogue` microservice & Memory Hog experiments on the `orders` microservice in a staggered manner.
+
+```
+kubectl apply -f utils/sample-chaos-injectors/chaos-experiments/catalogue/catalogue-pod-cpu-hog.yaml
+```
+
+Wait for ~60s
+
+```
+kubectl apply -f utils/sample-chaos-injectors/chaos-experiments/orders/orders-pod-memory-hog.yaml
+```
+
+Wait for ~60s
+
+```
+kubectl apply -f utils/sample-chaos-injectors/chaos-experiments/catalogue/catalogue-node-cpu-hog.yaml
+```
+
+Wait for ~60s
+
+```
+kubectl apply -f utils/sample-chaos-injectors/chaos-experiments/orders/orders-node-memory-hog.yaml
+```
+
+- Verify execution of chaos experiments
+
+  ```
+  kubectl describe chaosengine catalogue-pod-cpu-hog -n litmus
+  kubectl describe chaosengine orders-pod-memory-hog -n litmus
+  kubectl describe chaosengine catalogue-node-cpu-hog -n litmus
+  kubectl describe chaosengine orders-node-memory-hog -n litmus
+  ```
+
+### Step-5: Visualize Chaos Impact
+
+- Observe the impact of chaos injection through increased Latency & reduced QPS (queries per second) on the microservices
+  under test.
+
+  ![image](https://github.com/litmuschaos/litmus/blob/master/monitoring/screenshots/Sock-Shop-Dashboard.png?raw=true)
+
+  ![image](https://github.com/litmuschaos/litmus/blob/master/monitoring/screenshots/Node-and-Pod-metrics-Dashboard.png?raw=true)
+
+### Step-6 (optional): Inject continous chaos using Argo CD.
+
+- Install Argo workflow infrastructure.
+
+  - Create argo namespace
+
+    ```
+    kubectl create ns argo
+    ```
+
+  - Create the CRDs, workflow controller deployment with associated RBAC.
+
+    ```
+    kubectl apply -f https://raw.githubusercontent.com/argoproj/argo/stable/manifests/install.yaml -n argo
+    ```
+
+  - Install the argo CLI on the test harness machine (where the kubeconfig is available)
+
+    ```
+    # Download the binary
+    curl -sLO https://github.com/argoproj/argo/releases/download/v2.11.0/argo-linux-amd64.gz
+
+    # Unzip
+    gunzip argo-linux-amd64.gz
+
+    # Make binary executable
+    chmod +x argo-linux-amd64
+
+    # Move binary to path
+    mv ./argo-linux-amd64 /usr/local/bin/argo
+
+    # Test installation
+    argo version
+    ```
+
+- Create the Argo Access ServiceAccount
+
+  ```
+  kubectl apply -f https://raw.githubusercontent.com/litmuschaos/chaos-workflows/master/Argo/argo-access.yaml -n litmus
+  ```
+
+- Run litmuschaos experiments as Argo workflows using argo CLI or kubectl.
+
+  ```
+  argo cron create utils/sample-chaos-injectors/chaos-workflows-with-argo-CD/catalogue/catalogue-node-cpu-hog-workflow.yaml -n litmus
```

Comment:
We can separate/clearly lay the options - user may end up executing each of these commands in sequence

Reply:
Okay I will make these as a selection..